backend/compact-connect/lambdas/python/common/cc_common/data_model/schema/privilege/record.py

```diff
@@ -168,6 +186,11 @@ def generate_pk_sk(self, in_data, **kwargs):  # noqa: ARG001 unused-argument
         )
         return in_data
 
+    @validates_schema
+    def validate_deactivation_details_present_if_deactivation_update(self, data, **kwargs):  # noqa: ARG002 unused-argument
+        if data['updateType'] == UpdateCategory.DEACTIVATION.value and not data.get('deactivationDetails'):
```

Comment:
Per Justin's comment, can we just do this?

 ```suggestion
        if data['updateType'] == UpdateCategory.DEACTIVATION and not data.get('deactivationDetails'):
```

Reply:
When I initially added this schema validation with some tests to verify an error would be thrown, the equality check without the `.value` resulted in the error not being thrown. But strangely, I can't reproduce that issue now in the tests even after removing the `.value` so I believe you are both correct that this is not needed. I'll remove it.